### PR TITLE
Add semantic slider input component

### PR DESCRIPTION
**Pull request by krystian8207**
#57 issue solution.

### CODE REVIEW

File: examples/slider_input/app.R [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      uiOutput("slider", width = "200px"),
+      br(),
+      textOutput("val")
+    )
+  )
+}
+
+server <- shinyServer(function(input, output) {
+  output$slider <- renderUI({
+    shiny.semantic::sliderInput("my_slider", 0, 10, 5, color = "red")
```
Comment (dokato) [inline examples/slider_input/app.R]: should be `slider_input`

File: examples/slider_input/app.R [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+ui <- function() {
+  shinyUI(
+    semanticPage(
+      title = "Dropdown example",
+      suppressDependencies("bootstrap"),
+      suppressDependencies("semantic"),
+      tags$head(
+        tags$script(src = "https://rawgit.com/gdaunton/Semantic-UI/range-slider-build/dist/semantic.js"),
+        tags$link(rel="stylesheet", type="text/css", href="https://rawgit.com/gdaunton/Semantic-UI/range-slider-build/dist/semantic.css")
```
Comment (dokato) [inline examples/slider_input/app.R]: Here I think it should be done somehow within `get_dependencies`.

File: R/slider_input.R [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#' @export
+slider_input <- function(name, min, max, value, step = 0.01, n_ticks = 5, color = "") {
+  if (!(color %in% c("", "red", "yellow", "orange", "olive", "green", "teal", "blue", "violet", "purple", "pink", "brown", "grey"))) {
```
Comment (dokato) [inline R/slider_input.R]: Don't we have any constant `allowed_colours`? If not we should.

File: R/slider_input.R [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    stop("Wrong color parameter specified!")
+  }
```
Comment (dokato) [inline R/slider_input.R]: + actually that should be separate function

File: R/semanticPage.R [hunk 5]
```diff
@@ -19,7 +19,9 @@ get_dependencies <- function() {
                               c(file = file),
                               script = javascript_file,
                               stylesheet = css_files
-    )
+    ),
+    htmltools::tags$link(href = file.path(system.file("www", package = "shiny.semantic"), "range.css"), rel = "stylesheet"),
+    htmltools::tags$script(src = file.path(system.file("www", package = "shiny.semantic"), "range.js"))
```
Comment (dokato) [inline R/semanticPage.R]: Lines are too long. Could you break them?

File: examples/slider_input/app.R [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+server <- shinyServer(function(input, output) {
+  output$slider <- renderUI({
+    shiny.semantic::sliderInput("my_slider", 0, 10, 5, color = "red")
+  })
+  output$val <- renderPrint({
+    input$my_slider
+  })
+})
+
+shinyApp(ui = ui(), server = server)
```
Comment (dokato) [inline examples/slider_input/app.R]: No brackets in `ui`: `shinyApp(ui, server)`

File: examples/slider_input/app.R [hunk 7]
```diff
@@ -0,0 +1,30 @@
+library(shiny)
+library(shiny.semantic)
+
+ui <- function() {
+  shinyUI(
+    semanticPage(
+      title = "Dropdown example",
```
Comment (dokato) [inline examples/slider_input/app.R]: It's slider example.

File: R/slider_input.R [hunk 8]
```diff
@@ -0,0 +1,37 @@
+#' @export
```
Comment (dokato) [inline R/slider_input.R]: We need some documentation here.

File: examples/slider_input/app.R (hunk 6 above)
Comment (krystian8207) [reply examples/slider_input/app.R]: ui is prepared as function, see line 4

**Pull request merged**